drivers/wireless/ieee80211/bcm43xxx/bcmf_sdio.c

```diff
@@ -1065,10 +1065,20 @@ struct bcmf_sdio_frame *bcmf_sdio_allocate_frame(FAR struct bcmf_dev_s *priv,
           DEBUGPANIC();
         }
 
-      if ((entry = bcmf_dqueue_pop_tail(&sbus->free_queue)) != NULL)
+      if (!tx ||
+          sbus->tx_queue_count <
```

Comment:
do we need set to zero `tx_queue_count` during init, or `struct bcmf_sdio_dev_s` use `zalloc` / static allocation?

Reply:
tx_queue_count has been initialized to 0
https://github.com/apache/incubator-nuttx/blob/master/drivers/wireless/ieee80211/bcm43xxx/bcmf_sdio.c#L816